typings/browser.d.ts

```diff
@@ -1 +0,0 @@
-/// <reference path="browser/ambient/node/index.d.ts" />
```

Comment:
Typings files appear to have been deleted. This will need to be regenerated so that TypeScript users may continue to use remap-istanbul.